core/src/gui/main_window.cpp

```diff
@@ -489,12 +483,11 @@ void MainWindow::draw() {
         ImGui::SetColumnWidth(1, winSize.x - menuWidth - 60);
         ImGui::SetColumnWidth(2, 60);
         ImGui::BeginChild("Left Column");
-        float menuColumnWidth = ImGui::GetContentRegionAvailWidth();
 
         if (gui::menu.draw(firstMenuRender)) {
             core::configManager.acquire();
             json arr = json::array();
-            for (int i = 0; i < gui::menu.order.size(); i++) {
+            for (size_t i = 0; i < gui::menu.order.size(); i++) {
```

Comment:
Same issue as above, no reason to use a size_t here, if your menu is over 2 billion elements in length you have other issues